server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -1362,8 +1362,8 @@ boolean canSuspendTablets() {
     String address = sa.address.toString();
     UUID uuid = sld.getServerUUID(ThriftService.MANAGER);
     ServiceDescriptors descriptors = new ServiceDescriptors();
-    for (ThriftService svc : new ThriftService[] {ThriftService.MANAGER,
-        ThriftService.COORDINATOR}) {
+    for (ThriftService svc : new ThriftService[] {ThriftService.MANAGER, ThriftService.COORDINATOR,
```

Comment:
This updates the lock data with the services that the Manager handles after the Manager is fully up.